test/e2e/tests/metrics/errors.spec.js

```diff
@@ -317,12 +317,10 @@ describe('Sentry errors', function () {
           const [mockedRequest] = await mockedEndpoint.getSeenRequests();
           const mockTextBody = (await mockedRequest.body.getText()).split('\n');
           const mockJsonBody = JSON.parse(mockTextBody[2]);
-          const { level } = mockJsonBody;
-          const [{ type, value }] = mockJsonBody.exception.values;
           // Verify request
-          assert.equal(type, 'TypeError');
-          assert(value.includes(migrationError));
-          assert.equal(level, 'error');
+          assert(
+            JSON.stringify(mockJsonBody.exception).includes(migrationError),
+          );
```

Comment:
This assertion would probably just resolve to true or false, providing little debugging information upon failure.

Perhaps we could use` assert.match` instead:

```suggestion
          assert.match(
            JSON.stringify(mockJsonBody.exception),
            RegExp(migrationError).
          );
```

Reply:
Given your explanation of how the test mock works, this is no longer important